tests/BasicTest.cpp

```diff
@@ -45,8 +45,8 @@ int main()
   }
   catch (const sqlpp::postgresql::broken_connection& ex)
   {
-    std::cout << "Got exception: " << ex.what();
-    if (ex.what() != std::string("PostgreSQL error: failed to connect to the database"))
+    std::cout << "Got exception: '" << ex.what() << "'";
+    if (! strcmp(ex.what(), "Ident authentication failed for user \"unknown_user_must_fail\""))
```

Comment:
Postgresql errors are translated (so on different system language this will fail) remove this comparison. I think sqlpp::postgresql::broken_connection exception is precise enough. I didn't noticed it before, sory for that